### PR TITLE
Fix bug with `disk-quota` output parsing in `mila code`

### PR DESCRIPTION
**Pull request by lebrice**

### CODE REVIEW

File: milatools/cli/commands.py [hunk 1]
```diff
@@ -1034,32 +1034,49 @@ def _get_disk_quota_usage(
...
+    ```bash
+    #!/bin/sh
+    # (disk-quota)
+    echo "==== HOME ===="
+    lfs quota -h -u "${1:-$USER}" /home/mila
+
+    echo ""
+    echo "==== SCRATCH ===="
+    beegfs-ctl --cfgFile=/etc/beegfs/scratch.d/beegfs-client.conf --getquota --uid "${1:-$USER}"
+    ```
```
Comment (breuleux) [inline milatools/cli/commands.py:1047]: That's the content of the `disk-quota` command, remove.

File: milatools/cli/commands.py [hunk 2]
```diff
@@ -1034,32 +1034,49 @@ def _get_disk_quota_usage(
...
 
-          user/group     ||           size          ||    chunk files
-         name     |  id  ||    used    |    hard    ||  used   |  hard
-    --------------|------||------------|------------||---------|---------
-          normandf|1471600598||   97.20 GiB|  100.00 GiB||   806898|  1000000
+    ```bash
+    $ lfs quota -h -u "${1:-$USER}" /home/mila
```
Comment (breuleux) [inline milatools/cli/commands.py:1050]: We're not passing -h.

**Review by breuleux — COMMENTED** [review]
The call to `check_disk_quota` in `code` should be wrapped in a try/catch, just to ensure that a breaking change (e.g. changing the filesystem) does not impede the functionality (since it is not strictly necessary in order for `mila code` to work).

File: milatools/cli/commands.py (hunk 2 above)
Comment (lebrice) [reply milatools/cli/commands.py]: Fixed in e3d7a90107f6a2e5f53741cc9425e354af5bbeed

File: milatools/cli/commands.py (hunk 1 above)
Comment (lebrice) [reply milatools/cli/commands.py]: Fixed in e3d7a90107f6a2e5f53741cc9425e354af5bbeed

**Comment by lebrice** [comment]
> The call to `check_disk_quota` in `code` should be wrapped in a try/catch, just to ensure that a breaking change (e.g. changing the filesystem) does not impede the functionality (since it is not strictly necessary in order for `mila code` to work).

Fixed in caff6cfba8bb3345114ef17bdf3950f7b42fbfcd

**Review by breuleux — APPROVED** [review]

**Pull request merged**